pkg/util/net.go

```diff
@@ -117,15 +116,6 @@ func RetryTCPConnection(ctx context.Context, host string, port string) bool {
 	return status
 }
 
-func CreateLocalhostListenerOnPort(port int) (tcpnet.Listener, error) {
```

Comment:
I think etcd needs to re-vendor microshift

Reply:
Done